ci/check-documentation.py

```diff
@@ -114,8 +114,25 @@ def complain(message):
             if md_path in index_child_md_links:
                 referenced_md_files.add(target_path)
 
-        # check for existence of links to README.md files of models and demos
+        # check <omz_dir> link validity
+
+        for link in sorted([link for link in doc_page.omz_references() if link.startswith('<omz_dir>')]):
+            file_path = Path(link.replace('<omz_dir>', str(OMZ_ROOT)))
+
+            try:
+                file_relative_path = file_path.relative_to(OMZ_ROOT)
+            except ValueError:
+                complain(f'{md_path_rel}: invalid OMZ reference {file_path!r}')
+                continue
+
+            if str(file_relative_path) == md_path_rel: # self-link
+                continue
 
+            if not (file_path.is_file() or file_path.is_dir()):
+                complain(f'{md_path_rel}: OMZ reference "{file_relative_path}" target'
```

Comment:
```suggestion
                complain(f'{md_path_rel}: OMZ reference "{link}" target'
```

`file_relative_path` is not the reference - `link` is.